### ISSUE TITLE
How to prepare fine-tuning data for the temporal grounding task?

### ISSUE THREAD

**Issue by RayTang88** [DAMO-NLP-SG/VideoLLaMA3#54]
I noticed that you use the YouCook2 dataset for temporal grounding. When I prepare the data in this format, I modify the stage4_2b.sh script and replace the data_path with the YouCook2 jsonl file for fine-tuning. However, I encountered an issue where the process fails to start due to the lack of data_dict["conversations"]. It seems that the YouCook2 dataset doesn't have it. Could you please clarify how the data format for temporal grounding is handled on your side, especially how the temporal information is incorporated into the labels?

**Comment by nopa607** [comment]
training format is listed in README.
formatting the training data like that, and stagexx.sh runs successfully.

```[
    {
        "image": ["images/xxx.jpg"],
        "conversations": [
            {
                "from": "human",
                "value": "<image>\nWhat are the colors of the bus in the image?"
            },
            {
                "from": "gpt",
                "value": "The bus in the image is white and red."
            },
            ...
        ]
    },
    {
        "video": ["videos/xxx.mp4"],
        "conversations": [
            {
                "from": "human",
                "value": "<video>\nWhat are the main activities that take place in the video?"
            },
            {
                "from": "gpt",
                "value": "The main activities that take place in the video are the preparation of camera equipment by a man, a group of men riding a helicopter, and a man sailing a boat through the water."
            },
            ...
        ]
    },
    ...
]```

**Comment by RayTang88** [comment]
Thank you for your reply. These are two parts of a video data that I cut from the youcook2 dataset.
>
xHr8X2Wpmno_0 | https://www.youtube.com/watch?v=xHr8X2Wpmno | 226 | [ 47, 60 ] | pick the ends off the verdalago | val/xHr8X2Wpmno_0.mp4 | xHr8X2Wpmno
xHr8X2Wpmno_1 | https://www.youtube.com/watch?v=xHr8X2Wpmno | 226 | [ 67, 89 ] | combine lemon juice sumac garlic salt and oil in a bowl | val/xHr8X2Wpmno_1.mp4 | xHr8X2Wpmno

According to the data set you provided, the data set produced is as follows

{
"video": ["videos/xHr8X2Wpmno_0.mp4"],
"conversations": [
{
"from": "human",
"value": "<video>\nWhat are the main activities that take place in the video，please describe them with their corresponding start and end time point ?"
},
{
"from": "gpt",
"value": "47-60，pick the ends off the verdalago；67-89，combine lemon juice sumac garlic salt and oil in a bowl"
},
...
]
},
Is this a good training set? Or have a better idea
